tools/drdeploy.c

```diff
@@ -265,8 +265,8 @@ const char *options_list_str =
     "                          wait.  A value of 0 means don't wait for nudges to\n"
     "                          complete."
 #    else  /* WINDOWS */
-    /* FIXME i#840: integrate nudgeunix into drconfig on Unix */
-    "Note: please use the nudgeunix tool to nudge processes on Unix.\n";
+    /* FIXME i#840: integrate drnudgeunix into drconfig on Unix */
```

Comment:
Ah here's an already-filed issue on my top-level comment: ok let's leave it for that issue.
No action requested.  (Githhub should add a type of comment that is already resolved.)